src/base/Utils.js

```diff
@@ -146,6 +146,10 @@ export function capitalize(text) {
     return text.charAt(0).toUpperCase() + text.slice(1);
 }
 
+export function pluralize(count, text) {
```

Comment:
Where do you plan to use it? I can see the cases where you want to skip the count if it's 1, and when you want to say "one" instead of "1". Cool function otherwise.